chainer/distributions/bernoulli.py

```diff
@@ -5,6 +5,7 @@
 from chainer.backends import cuda
 from chainer import distribution
 import chainer.distributions.utils
+from chainer.functions import sum_to
```

Comment:
We do not allow importing non-module objects: https://docs.openstack.org/hacking/latest/user/hacking.html#imports

Please import like `from chainer.functions.math import sum` and use `sum.sum_to`.